### ISSUE TITLE
Eclipse Foundation removing PHP support from eclipse.dev

### ISSUE THREAD

**Issue by pshipton** [eclipse-openj9/openj9-docs#1344]
The Eclipse Foundation has announced the following. I don't see much or any use of php for the docs, although there are some links in the footer, and a couple in other content, that we should perhaps update. Since they refer to eclipse.org there may be no updates required.

https://www.eclipse.org/legal/privacy.php
https://www.eclipse.org/legal/termsofuse.php
https://www.eclipse.org/legal/copyright.php

https://www.eclipse.org/eclipse/debug/index.php
https://www.eclipse.org/org/documents/social_media_guidelines.php
https://www.eclipse.org/security/policy.php

We [Eclipse Foundation] are going to remove PHP support from eclipse.dev in December 2024.  This change only affects those projects that use eclipse.dev to host their project websites, and that have not already moved to either our Hugo based templates or another content generation technology.  

We will be doing an audit of eclipse.dev and filing helpdesk tickets over the summer to help track this change for affected projects.  However you don't need to wait for us to open those tickets if you'd like to get this done sooner rather than later.

**Comment by Sreekala-Gopakumar** [comment]
@pshipton - There are no instances of https://www.eclipse.dev in the documents. So, that means no changes are required in the documents, right?

Thanks!

**Comment by pshipton** [comment]
Yes, could be. We'll just monitor and see if Eclipse opens any issues against OpenJ9.
